jobserver/models/outputs.py

```diff
@@ -95,7 +104,7 @@ def get_absolute_url(self):
                 "project_slug": self.release.workspace.project.slug,
                 "workspace_slug": self.release.workspace.name,
                 "pk": self.release.id,
-                "path": self.path,
+                "path": self.name,
```

Comment:
This was a driveby bugfix - `self.path` is the path to the file on disk (relative to `RELEASES_STORAGE`), and shouldn't be part of a url.

`self.name` is the users name for the file (e.g. "output/file1.txt"), and is the part that should be used in the url

Reply:
As an aside, this is why I like hardcoding url expectations in tests, rather than asserting the output matchs a reverse lookup. That tests would have have passed, but the url is wrong.

Hardcoding url means we are encoding our url model outside of just urls.py, which is good to catch these errors. It does mean that when you change urls, there's some updating to do, but I think it comes on the right side of cost/benefit for me.